platform.txt

```diff
@@ -50,10 +50,10 @@ compiler.define=-DARDUINO=
 
 # this can be overriden in boards.txt
 build.extra_flags=
-
+build.lfclk_flags=
 nrf.sdk.path={runtime.platform.path}/cores/nRF5/SDK
 
-compiler.nrf.flags=-DNRF5 "-I{nrf.sdk.path}/components/toolchain/" "-I{nrf.sdk.path}/components/toolchain/CMSIS/Include" "-I{nrf.sdk.path}/components/toolchain/gcc/" "-I{nrf.sdk.path}/components/device/" "-I{nrf.sdk.path}/components/drivers_nrf/delay/" "-I{nrf.sdk.path}/components/softdevice/{softdevice}/headers/"
+compiler.nrf.flags=-DNRF5 "-I{nrf.sdk.path}/components/toolchain/" "-I{nrf.sdk.path}/components/toolchain/CMSIS/Include" "-I{nrf.sdk.path}/components/toolchain/gcc/" "-I{nrf.sdk.path}/components/drivers_nrf/hal/" "-I{nrf.sdk.path}/components/device/" "-I{nrf.sdk.path}/components/softdevice/{softdevice}/headers/"
```

Comment:
Is this a merge conflict? I see a `components/drivers_nrf/delay/` folder in the repo but not `components/drivers_nrf/hal/` ... the delay is needed now.